src/interp/InterpreterStatements.ml

```diff
@@ -728,6 +724,7 @@ let eval_transparent_function_call_symbolic_inst (span : Meta.span)
                      we also need to update the generics.
                   *)
                   let func = FunId fid in
+                  (* TODO: the `trait_method_generics` look fishy *)
```

Comment:
@sonmarcho What are the `trait_method_generics` supposed to correspond to? From the name I'd expect these would be the generics _just for the method_, but here you pass generics valid for the function item (which include impl generics).